### PR TITLE
[IMP] pos_hr: follow-up on login screen changes

### PR DESCRIPTION
**Pull request by adgu-odoo**
This commit follows what have been done in
https://github.com/odoo/odoo/pull/165956. It adapts the date and time format on the login screen. It also removes the unused images.

task-id: 4047473

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/pos_hr/static/src/app/login_screen/login_screen.js [hunk 1]
```diff
@@ -24,6 +25,17 @@ export class LoginScreen extends Component {
             },
             exclusive: true, // takes exclusive control on the barcode reader
         });
+        this.timeFormat = localization.timeFormat;
```
Comment (vlst-odoo) [inline addons/pos_hr/static/src/app/login_screen/login_screen.js:28]: could you extract all of this into a hook?
something like
```js
this.time = useTime();
```
and in the xml you would have `t-esc="time.day"` or `t-esc="time.hours"`

File: addons/pos_hr/static/src/app/login_screen/login_screen.js [hunk 2]
```diff
@@ -24,6 +25,17 @@ export class LoginScreen extends Component {
...
+        let dateFormat = localization.dateFormat;
+        const splitDateFormat = dateFormat.split(/[^a-zA-Z]+/);
+        for (const format of splitDateFormat) {
+            if (format === "MM") {
+                dateFormat = dateFormat.replace("MM", "LLLL");
+            } else if (format === "yy") {
+                dateFormat = dateFormat.replace("yy", "yyyy");
+            }
+        }
+        this.dateFormat = dateFormat.replace(/[^a-zA-Z]+/g, ", ");
```
Comment (vlst-odoo) [inline addons/pos_hr/static/src/app/login_screen/login_screen.js:38]: the reason for this change is to have the date before the month or the other way around, depending on the localization, right?

In that case, do you think we could simplify this ? Something like
```js
this.dateFormat = localization.dateFormat
    .replace(/MM/g, "LLLL")  
    .replace(/\/yy$/, "/yyyy")
    .replace(/[^a-zA-Z]+/g, ", ");
```

Comment (adgu-odoo) [reply addons/pos_hr/static/src/app/login_screen/login_screen.js:38]: That would indeed be better I'll try that

**Review by vlst-odoo — APPROVED** [review]
robodoo r+

**Pull request closed without merging**